src/main/java/mate/academy/internetshop/dao/jdbc/UserDaoJdbcImpl.java

```diff
@@ -0,0 +1,172 @@
+package mate.academy.internetshop.dao.jdbc;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Optional;
+import java.util.Set;
+import mate.academy.internetshop.dao.UserDao;
+import mate.academy.internetshop.exceptions.DataProcessingException;
+import mate.academy.internetshop.lib.Dao;
+import mate.academy.internetshop.model.Role;
+import mate.academy.internetshop.model.User;
+import mate.academy.internetshop.util.ConnectionUtil;
+
+@Dao
+public class UserDaoJdbcImpl implements UserDao {
+    @Override
+    public Optional<User> findByLogin(String login) {
+        String query = "SELECT * FROM users WHERE login=?;";
+        try (Connection connection = ConnectionUtil.getConnection()) {
+            PreparedStatement statement = connection.prepareStatement(query);
+            statement.setString(1, login);
+            ResultSet resultSet = statement.executeQuery();
+            if (resultSet.next()) {
+                return Optional.of(getUserFromResultSet(resultSet));
+            }
+            return Optional.empty();
+        } catch (SQLException e) {
+            throw new DataProcessingException("find user are failed", e);
+        }
+    }
+
+    @Override
+    public User create(User user) {
+        String query = "INSERT INTO internet_shop.users (login, password) VALUES (?,?);";
```

Comment:
```suggestion
        String query = "INSERT INTO users (login, password) VALUES (?,?);";
```
прибери пліз використання імені схеми з SQL запитів. Схему ми конфігуруємо при формуванні коннекшину в класі ConnectionUtil

Reply:
Done